ddtrace/contrib/pytest/_efd_utils.py

```diff
@@ -32,7 +33,6 @@ class __EFD_RETRY_OUTCOMES:
     EFD_FINAL_FLAKY = "dd_efd_final_flaky"
 
 
-_EFD_RETRY_OUTCOMES = __EFD_RETRY_OUTCOMES()
 _EFD_FLAKY_OUTCOME = "Flaky"
```

Comment:
Is this the capitalized `Flaky` that shows up in the final stats? If so, we may want:
```suggestion
_EFD_FLAKY_OUTCOME = "flaky"
```